02.calendar/calendar.js

```diff
@@ -0,0 +1,19 @@
+const argv = require("minimist")(process.argv.slice(2));
+
+const today = new Date();
+const year = argv["y"] || today.getFullYear();
+const month = argv["m"] || today.getMonth() + 1;
+
+const firstDay = new Date(year, month - 1, 1);
+const lastDay = new Date(year, month, 0);
```

Comment:
```suggestion
const firstDate = new Date(year, month - 1, 1);
const lastDate = new Date(year, month, 0);
```

dayは曜日や年月日の日を表すことが多いので、dateの方が適してそうです。

Reply:
アドバイスありがとうございます🙏
Dateに修正しました！